spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin/DiscoveryClientEndpointLocatorTest.java

```diff
@@ -31,26 +34,46 @@
 
 import zipkin.Endpoint;
 
-import static org.assertj.core.api.BDDAssertions.then;
-import static org.mockito.BDDMockito.given;
-
 /**
  * @author Marcin Grzejszczak
  */
 @RunWith(MockitoJUnitRunner.class)
 public class DiscoveryClientEndpointLocatorTest {
 
 	@Mock DiscoveryClient discoveryClient;
+	@Mock EndpointCache endpointCache;
+
 	@InjectMocks DiscoveryClientEndpointLocator discoveryClientEndpointLocator;
 
+    @Before
```

Comment:
I started using eclipse formatting plugin 4 idea at the end, it seem validation does not include tests:)